src/ts/cosmosJourneyer.ts

```diff
@@ -314,6 +314,8 @@ export class CosmosJourneyer {
         await starSystemView.initAssets();
         starSystemView.resetPlayer();
 
+        await alertModal("Loading Complete! Welcome to Cosmos Journeyer."); 
```

Comment:
Yup that's the right place! But you still need to check for `navigator.keyboard` in order to only display the modal with the message about the QWERTY layout being assumed by default when it is not supported.